Tasks/AzureFunctionAppV1/deploymentProvider/BuiltInLinuxWebAppDeploymentProvider.ts

```diff
@@ -14,7 +14,18 @@ const premiumPlanRunsFromPackage: string = ' -WEBSITE_RUN_FROM_PACKAGE true';
 const linuxFunctionRuntimeSettingValue = new Map([
     [ 'DOCKER|microsoft/azure-functions-dotnet-core2.0:2.0', 'dotnet ' ],
     [ 'DOCKER|microsoft/azure-functions-node8:2.0', 'node ' ],
-    [ 'DOCKER|microsoft/azure-functions-python3.6:2.0', 'python ']
+    [ 'DOCKER|microsoft/azure-functions-python3.6:2.0', 'python '],
```

Comment:
this needs to be updated in AzureRmWebAppDeploy task as well.

Reply:
done